pokemongo_bot/__init__.py

```diff
@@ -42,8 +43,22 @@ def take_step(self):
         location = self.navigator.take_step()
         cells = self.find_close_cells(*self.position[0:2])
 
+        cells = self.find_close_cells(*location)
```

Comment:
I just stumbled over this PR and I'm not able to test it myself right now, but shouldn't it be a concatenation between the cells from line 44 (new) and line 46 (new) instead of a replacement?